resources/views/auth/passwords/reset.blade.php

```diff
@@ -15,12 +15,16 @@
                         <h4>{{ trans('auth.validation.issues') }}</h4>
                         <ul class="list -compacted">
                             @foreach ($errors->all() as $error)
-                                <li>{{ $error }}</li>
+                                <li>
+                                    {{ $error }} 
+                                    @if (strpos($error, 'create-password'))
+                                        <a href="/password/reset">Get a new password link</a>
```

Comment:
I'm gonna double check on this copy, what we have in the ticket differs from the designs and I'm not sure which is newer!

Reply:
@Dinnall turns out this is the outdated copy, sorry about that! Let's switch this to use "Resend email to create/reset password" instead.

```suggestion
                                        <a href="/password/reset">Resend email to create/reset password</a>
```

Reply:
I just saw Leah's comment and I pushed the suggested change.

Reply:
Hmm I'm still seeing "Get a new password link" on the PR 🤔

Reply:
Pushed again